paperqa/agents/env.py

```diff
@@ -33,8 +39,20 @@
 
 POPULATE_FROM_SETTINGS = None
 
+DEFAULT_TOOL_NAMES: list[str] = [
```

Comment:
I like this, nice.

What do you think of moving this to be co-located with `AVAILABLE_TOOL_NAME_TO_CLASS` in `tools`?